test/prog/dftb+/bin/autotest2

```diff
@@ -196,12 +196,12 @@ case "${STAGE_SEP}$stages" in
 esac
 
 
-# Check if executable is there:
-if [ ! -x "${app_cmd}" -a -n "${STAGE_RUN}" ]; then
-  echo "${SCRIPTNAME}: Error: Executable '${app_cmd}' does not exist" 1>&2
-  echo "Type '${SCRIPTNAME} -h' for help" 1>&2
-  exit 1
-fi
+## Check if executable is there:
```

Comment:
Any point in saving this and having a flag to turn the check on/off for use with the program instead of the API?

Reply:
Done in 2f89d8bb